app/controllers/routeDetails/transit/OfficeOfTransitETAController.scala

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.routeDetails.transit
+
+import controllers.actions._
+import controllers.{NavigatorOps, SettableOps, SettableOpsRunner}
+import forms.DateTimeFormProvider
+import models.{Index, LocalReferenceNumber, Mode}
+import navigation.Navigator
+import navigation.annotations.Transit
+import pages.routeDetails.transit.{OfficeOfTransitCountryPage, OfficeOfTransitETAPage, OfficeOfTransitPage}
+import play.api.i18n.{I18nSupport, MessagesApi}
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import repositories.SessionRepository
+import uk.gov.hmrc.play.bootstrap.frontend.controller.FrontendBaseController
+import views.html.routeDetails.transit.OfficeOfTransitETAView
+import javax.inject.Inject
+import models.reference.{Country, CustomsOffice}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class OfficeOfTransitETAController @Inject() (
+  override val messagesApi: MessagesApi,
+  implicit val sessionRepository: SessionRepository,
+  @Transit implicit val navigator: Navigator,
```

Comment:
As this is an indexed page it will need to use the office of country navigator provider (see the other indexed controllers for reference)

Reply:
That's what I'm doing now. Just putting this code in then adding the navigation to all the pages